test/extras/no_use_pycon.py

```diff
@@ -5,9 +5,8 @@
 #
 # This test tests for SNOW-186747
 
-import requests
-
 import snowflake.connector  # NOQA
+from snowflake.connector.vendored import requests
```

Comment:
This test makes sure that the monkey patching didn't affect the use cases where our connector was imported, but it never used. This has burned us in the past.
Please don't change it even though the monkey patching should really not affect other libraries anymore it won't hurt to have this one extra test for a bit longer 😄